Author: Andreas Tille <tille@debian.org>
Last-Updated: Tue, 08 Mar 2016 11:15:04 +0100
Description: Remove RPATH
 see https://wiki.debian.org/RpathIssue for details

--- a/libxrdp/Makefile.am
+++ b/libxrdp/Makefile.am
@@ -19,7 +19,7 @@ endif
 if XRDP_TJPEG
 EXTRA_DEFINES += -DXRDP_JPEG -DXRDP_TJPEG
 EXTRA_INCLUDES += -I/opt/libjpeg-turbo/include
-EXTRA_FLAGS += -L/opt/libjpeg-turbo/lib -Wl,-rpath -Wl,/opt/libjpeg-turbo/lib
+EXTRA_FLAGS += -L/opt/libjpeg-turbo/lib
 EXTRA_LIBS += -lturbojpeg
 else
 if XRDP_JPEG
@@ -30,7 +30,7 @@ endif
 
 if GOT_PREFIX
 EXTRA_INCLUDES += -I$(prefix)/include
-EXTRA_FLAGS += -L$(prefix)/lib -Wl,-rpath -Wl,$(prefix)/lib
+EXTRA_FLAGS += -L$(prefix)/lib
 endif
 
 AM_CFLAGS = \
--- a/xrdp/Makefile.am
+++ b/xrdp/Makefile.am
@@ -12,7 +12,7 @@ endif
 
 if GOT_PREFIX
 EXTRA_INCLUDES += -I$(prefix)/include
-EXTRA_FLAGS += -L$(prefix)/lib -Wl,-rpath -Wl,$(prefix)/lib
+EXTRA_FLAGS += -L$(prefix)/lib
 endif
 
 AM_CFLAGS = \
